### ISSUE TITLE
Fabric 1.20.4 support

### ISSUE THREAD

**Issue by Zorklis** [BeansGalaxy/Beans-Backpacks-2#80]

**Comment by BeansGalaxy** [comment]
Likely not until the 1.20.1 version is out of beta since 20.1 is the most used version. 1.21 will likely see an update before 20.4 since they are making big changes to item data with the removal of NBT and I'm excited to work with their new system when it's released.

**Comment by Zorklis** [comment]
Then keep it open so people know it lacks 1.20.4 support and clearly isn't completed

**Comment by BeansGalaxy** [comment]
Good call, I also have a disclaimer in my ReadMe but I think my modrinth/forge descriptions link directly to the issues
